TypeScript: should `emit` and `off` be a part of `Broker` namespace?

`foo-foo-mq` version: 8.0.0
`typescript` version: 4.4.4

We have unit tests asserting that our code correctly handles events raised by `foo-foo-mq`. here is an example
```typescript
import * as sinon from 'sinon';
import * as rabbot from 'foo-foo-mq';
import { assert } from 'chai';
import { EventEmitter } from 'events';

describe('#registerEvents specific events checks', () => {
    let sandbox: sinon.SinonSandbox;
    let eventEmitter: EventEmitter;

    beforeEach(() => {
      sandbox = sinon.createSandbox();
      eventEmitter = new EventEmitter();
      
       // our code that responsds to foo-foo-mq events by re-emitting
      registerEvents(eventEmitter, 1, 5);
    });

    afterEach(() => {
      rabbot.off();
    });

    after(() => {
      sandbox.restore();
    });

    it('expected data returned from the failed event', (done) => {
      const failureData = {
        consecutiveFailures: 1,
      };
      const timeStampObj = new Date();

      eventEmitter.on('failed', (data) => {
        assert.deepEqual(data, {
          connectionAttempts: failureData.consecutiveFailures,
          attemptsRemaining: 4,
          remainingTimeToRetry: 1, // how to calculated value
          message:
            'Attempting to reconnect until time runs out or retries are expired',
        });
        done();
      });
      rabbot.emit('failed', failureData, timeStampObj);
    });
});
```

The problem is that `tsc` fails with `error TS2339: Property 'emit' does not exist on type 'typeof Broker'.`

Should `emit` be added to the `Broker` [type/namespace](https://github.com/Foo-Foo-MQ/foo-foo-mq/blob/a8781ba72d8c40f4cbc4a3a09bdff2fc61771327/src/index.d.ts#LL5C21-L5C21)? Or is there a better way to simulate a `foo-foo-mq` event?

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -92,6 +92,10 @@ declare namespace Broker {
 
   // Event handler
   export function on(event: string, handler: (...args: any[]) => void): any;
+  /** Remove all event subscriptions */
+  export function off(): void;
+  /** Emit an event*/
+  export function emit(topic: string, data?: any, timestamp?: Date): void;
 
   // Unhandled messages
   export function onUnhandled(handler: (msg: Message<any>) => void): void;

